### DISCUSSION TITLE
Video on De-embedding using negation elements

### DISCUSSION THREAD

**Discussion by jhillairet** [scikit-rf/scikit-rf#899] [General]
Found in the latest email from Microwaves 101:

> Often in Microwave engineering, you obtain measured S-parameter data on a device that is embedded in a network that you cannot take apart, so the measurement reference planes are not right at the device you are interested in. The hardware that you don't want in the measurement could be RF coax cables, or an evaluation board, or other test fixture that the part is mounted to.  The process of mathematically removing the network is called de-embedding. 
> 
> In this month's video, Anurag Bhargava explains [how to de-embed data](https://microwaves101.us9.list-manage.com/track/click?u=8bedd8abba89950614f02ffde&id=7c810b9d27&e=8e386cc95d) using Keysight's Advanced Design System (ADS).

https://www.microwaves101.com/encyclopedias/de-embedding-using-negation-elements#anurag

**Comment by mhuser** [comment]
Nice!
Scikit-rf offer the same kind of [s-parameters de-embedding (even 2N-ports)](https://scikit-rf.readthedocs.io/en/latest/examples/networktheory/Balanced%20Network%20De-embedding.html) of known fixtures by inverting the T-Matrix behind the hood, but in less user friendly 😅
